catalog/weather/forecast.js

```diff
@@ -21,30 +21,28 @@ function main(params) {
     var language = params.language || 'en-US';
     var units = params.units || 'm';
     var timePeriod = params.timePeriod || '10day';
-    var timeURL ='/forecast/daily/10day';
+    var url = 'https://twcservice.mybluemix.net/api/weather/v1/geocode/' + lat + '/' + lon;
```

Comment:
So'm confuse if they moved to a new API, why it went from `weather/v2` instead of `weather/v1` seems backward